pkg/eraser/eraser.go

```diff
@@ -205,14 +212,14 @@ func removeVulnerableImages() (err error) {
 	}
 
 	// TESTING :
-	r, err = listImages(backgroundContext, imageClient, "")
+	imageTest, err := c.listImages(backgroundContext, "")
 	if err != nil {
 		return err
 	}
 
-	var allImages2 []string
+	allImages2 := make([]string, len(allImages))
```

Comment:
```suggestion
allImages2 := make([]string, 0, len(allImages))
```

Otherwise we allocate a slice that is populated with empty strings.